sycl/test-e2e/Basic/auto_local_range.cpp

```diff
@@ -0,0 +1,67 @@
+// RUN: %{build} -I . -o %t.out
+// RUN: %{run} %t.out
+
+#include <cstdlib>
+#include <sycl/ext/oneapi/experimental/auto_local_range.hpp>
```

Comment:
Extensions (including experimental ones) are normally included as part of sycl/include/sycl/sycl.hpp.